ffi_internals/src/alias_resolution.rs

```diff
@@ -0,0 +1,311 @@
+//!
+//! This module provides methods for parsing aliases out of a module and writing their definitions
+//! to disk, so that when ffi_* macros encounter references to aliases defined in remote crates, the
```

Comment:
ok, super nitpick here, but thought I'd bring it up just in case it resonates... when reading Rust and I see `references`, I think `&thing`--I think one can assume that's not what you're talking about in most of these cases, but it might be nice to use a different term if you can, just for clarity's sake. Don't feel obligated to change as at this point it kinda just feels like something my dumb brain picks up on, but I wouldn't mind. 👯‍♂️

Reply:
Yeah good point, don't need to overload `reference`. 👍  I'll rephrase all of those.